### ISSUE TITLE
support select count(time) from ... or select count(1) from ...

### ISSUE THREAD

**Issue by jordens** [influxdb/influxdb#961]
It would be nice if time and sequence_number would become more similar to regular columns. E.g. it to be able to determine the number of rows in a series without knowing the names of the available columns. 

On a related note, "select time from series" is always empty.

**Comment by pnpritchard** [comment]
:+1: for time to be allowed in the SELECT clause and within functions like MIN

**Comment by Vanhoeyl** [comment]
+1 really need to have way to query the time field and do some basic math. ATM this is one of the missing features that we need before we can use influxd

**Comment by beckettsean** [comment]
Issue has drifted from the original request. Please open a new issue for functions/operators on time.

**Comment by jordens** [comment]
@beckettsean Has the original issue been adressed?

**Comment by beckettsean** [comment]
@jordens the original issue has not been addressed, but if it is still relevant a cleaner, newer issue using 0.9 examples is preferred. It is not possible to do something like `SELECT COUNT(*) FROM measurement`, it still requires that you know at least one field, e.g. `SELECT COUNT(value) FROM measurement`.

**Comment by pootzko** [comment]
Excuse me for my blatancy but why do you keep closing issues if they haven't been addressed..? Just because the version number of influx has been bumped by 0.1, doesn't mean the issue is gone. Hiding problems under the rug isn't really helping anyone.

**Comment by dandv** [comment]
Can we please reopen this issue exactly for what the OP asked in the first comment (minus the sequence_number bit)?

In other words, `SELECT COUNT(time) FROM measurement WHERE...` should return the maximum of the counts for each of the columns returned by `SELECT COUNT(*) FROM measurement WHERE...`.

**Comment by pauldix** [comment]
@dandv you can't `SELECT COUNT(time)` because it doesn't exist as a column. Every column keeps its own time so you have to specify some column name other than time. This is part of the lower level implementation and is very unlikely to change.

**Comment by brahma-dev** [comment]
Can we please reopen this or #8882.
